source/retryable-reads/tests/aggregate-serverErrors.yml

```diff
@@ -156,8 +156,7 @@ tests:
             data: { failCommands: [aggregate], errorCode: 10107 }
         operation:
             <<: *retryable_operation
-            result:
-                error: true
+            error: true
```

Comment:
Note this results in some odd .json. This `result` from the `<<: *retryable_operation` now sticks around and the json file has:
```
        "result": [
          {
            "_id": 2,
            "x": 22
          },
          {
            "_id": 3,
            "x": 33
          }
        ],
        "error": true
```

I'll let you decide if this is acceptable or not.